build.gradle

```diff
@@ -2,7 +2,7 @@
 
 buildscript {
     ext.kotlin_version = "1.3.41"
-    ext.lib_version = "6.0.1"
+    ext.lib_version = "6.1.0"
     ext.lib_group = "com.reach5.identity"
     ext.min_sdk_version = 19
     ext.target_sdk_version = 29
```

Comment:
we should increment the ext.version_code

Reply:
Shall I set the `ext.version_code` to 2?